CHANGES.rst

```diff
@@ -49,6 +51,8 @@ Cubeviz
   ``wavelength_unit`` (use ``value_unit``), ``show_wavelength`` (use ``show_value``),
   ``slice`` (use ``value``). [#2878]
 
+- Spectral Extraction: renamed ``collapse_to_spectrum(...)`` to ``extract(...)``. [#2859]
```

Comment:
There was a rebase conflict here, I kept this in (which it should be once background subtraction is merged). If I needs to be taken out, will do